.github/workflows/CI.yml

```diff
@@ -0,0 +1,64 @@
+name: Tests
+
+on:
+  pull_request: {}
```

Comment:
can you modify it to run gh workflow on develop branch on both pull_request, push?

Reply:
that makes sense